docs/blocks/custom-fluids.md

```diff
@@ -0,0 +1,851 @@
+          },
+
+          {
+            "condition": "query.block_neighbor_has_any_tag(0, 0, -1, 'template_full')",
+            "set_block_property": {
+              "fluids:nz": 1
+            }
+          },
+          {
+            "condition": "query.block_neighbor_has_any_tag(0, 0, 1, 'template_full')",
+            "set_block_property": {
+              "fluids:z": 1
+            }
+          },
+          {
+            "condition": "query.block_neighbor_has_any_tag(1, 0, 0, 'template_full')",
+            "set_block_property": {
+              "fluids:x": 1
+            }
+          },
+          {
+            "condition": "query.block_neighbor_has_any_tag(-1, 0, 0, 'template_full')",
+            "set_block_property": {
+              "fluids:nx": 1
+            }
+          },
+          {
+            "condition": "query.block_neighbor_has_any_tag(0, 1, 0, 'template_full')",
+            "set_block_property": {
+              "fluids:top": 1
+            }
+          },
+          {
+            "condition": "query.block_neighbor_has_any_tag(0, -1, 0, 'template_full')",
+            "set_block_property": {
+              "fluids:bottom": 1
+            }
+          },
+
+          {
+            "condition": "!query.block_neighbor_has_any_tag(0, 0, -1, 'template_full')",
+            "set_block_property": {
+              "fluids:nz": 0
+            }
+          },
+          {
+            "condition": "!query.block_neighbor_has_any_tag(0, 0, 1, 'template_full')",
+            "set_block_property": {
+              "fluids:z": 0
+            }
+          },
+          {
+            "condition": "!query.block_neighbor_has_any_tag(1, 0, 0, 'template_full')",
+            "set_block_property": {
+              "fluids:x": 0
+            }
+          },
+          {
+            "condition": "!query.block_neighbor_has_any_tag(-1, 0, 0, 'template_full')",
+            "set_block_property": {
+              "fluids:nx": 0
+            }
+          },
+          {
+            "condition": "!query.block_neighbor_has_any_tag(0, 1, 0, 'template_full')",
+            "set_block_property": {
+              "fluids:top": 0
+            }
+          },
+          {
+            "condition": "!query.block_neighbor_has_any_tag(0, -1, 0, 'template_full')",
+            "set_block_property": {
+              "fluids:bottom": 0
+            }
+          }
+        ]
+      },
+      "fluids:pick_up": {
+        "decrement_stack": {},
+        "run_command": {
+          "command": "give @p fluids:template_bucket"
+        },
+        "die": {}
+      }
+    }
+  }
+}
+```
+
+</Spoiler>
+
+## Outer Fluid Block 1
+
+Below is the JSON for the `outer fluid block 1`. To replicate the json, copy and quick replace `template` with your fluid's name. This block has the same function as the source block but it can't be picked up and it places `outer fluid block 2` instead.
+
+<CodeHeader>BH\blocks\fluid_template\fluid_template1.json</CodeHeader>
```

Comment:
Please make the codeheader follow the same style guide, `BP/blocks/filename.json`